internal/cli/apis.go

```diff
@@ -276,6 +316,8 @@ auth0 apis update <id|audience> --name myapi`,
 
 	apiName.RegisterStringU(cmd, &inputs.Name, "")
 	apiScopes.RegisterStringSliceU(cmd, &inputs.Scopes, nil)
+	apiAllowOfflineAccess.RegisterBool(cmd, &inputs.AllowOfflineAccess, true)
+	apiTokenLifetime.RegisterInt(cmd, &inputs.TokenLifetime, 0)
```

Comment:
```suggestion
	apiTokenLifetime.RegisterInt(cmd, &inputs.TokenLifetime, 86400)
```
`86400` (1 day) is the default value for new APIs